app.json

```diff
@@ -0,0 +1,13 @@
+{
+  "env": {
+    "PRIVATE_KEY": {
+      "description": "-----BEGIN RSA PRIVATE KEY-----\nMIIEogIBAAKCAQEAp/dfMg8IUFA5Y66zZm73W0qTQQdRLjrrADz1LYpVBwZBAlZW\nKhlY0aRq073XYK5xKDq8dQbLl/S5U6YBkc2OrSRe8A8+Su8UMynrYZ68bkLl4qg2\nA4pcwzdW9gad1mWKuU6sE21JnJKokpyDXwf7ZmFNvjJkMY9d0JCS1iU8w5I+xfwp\nn7Ryota74dfA9MvVuZFJefOdYeo1RYAu4QU8pnDlCf7q2PVUnhDGuStbUprM+T3N\nuqX43iNTDI4kQVWTRhs3f/vEtJAbN++qKZuKd66Ge4jh+a4/O6y3k2WFlr3u20uy\nOw9jBtHPlHAV6VpqQAW9L2B16TMbQ4QDMeJaswIDAQABAoIBADn6e5Ot+LKwXn74\nKNjs6NlMTrQdsHoCwGeeowsLOvpnwDzsTGXPxgbneDCK0TOAQ3MdgzvqEpDrWqVz\nIguQJvXBHjxDJr9680AN8kAKKQTudqfcZHioZAjGq2/VLNp6xn3cmC1EhipTvXiO\nKhK9kJvwboN+pXMuB7YckXL1+5mN7CtAllNUcc0n7D0gCWAMzLRIugiKMHMdTAQJ\nu07VS43YwzDqOw4NYG5mRiR68p2d1Pk7NMCzGfb9DvPfKvVTYXYHzYblM+vgWOQO\nGx+/gIAcrM8eWu+n+fM7iZP9CEuXSHUepUDwzIn4I6/GCu9iR2CCOigivfsVNlM/\nSYllsVECgYEA32dj0n0HyCoXrRkz9xRcLcfZhABnMOIoWsRfpBHLhgQVrlzjcTAg\nB2Ot4fCoo23wwW6srFZSph0WWiuTnkZ/7sQMqEPVINxsbXhqC3R4a+6tSLqbawsT\nBi988ZkEEizeNfc16gZPnHlxQ/3CWpfQMfiIsKR+mFSX75r7kj20mEUCgYEAwHlF\nSFttzT1/JkPILTBIZvP80hEkB616errSMdslaEn+f2v6AGmGN5IrzJpTyb6ADfNy\nCHll9xcRBi0ozFEsyvH8DQU2eUhIMw0M/wjZVVs0j+P5AbdqywQFnzDUD2/xvMd8\nwdnVGgNx6JaAKe6B7XgEi1PaCIRXnkQMLZc8ApcCgYAS2A27e4BkiehlWXLgrH75\n/ZmDYOVTYkA1J8tJc5bPPxj1FDC7I2wQM4A/RqVy45sjNcqq3r0p9dok0PgqqZew\n9Ch9KPuZQf6DXVzUioZ91CwZrmPdq5l3NBo95sJJJi+4LZoGm3GrqTK8oXL7YULW\ndDhTbgyfnMMUGuaNd1b8YQKBgDlMYNm3jiqJYi7MAu2g8AzmGZ+XxrgaZtnef/Cj\norE3RfI8/MHeqDwm4a8ZSlRGx+xfStVG/pneXg0OqJSandEU1MozgRH9+LBp9DKA\nOeL1MeKcJWbjaYFigyHu8VJfKwHeCT/UT2JdIC3zj9frzvYYwpGPttrOOx88druj\nyBOlAoGAYCvlSNLd9Eu0yyXshKrL/b8F13M8Pgwq9XAZNC2ank52HHU/HGQwymjt\nERm6kRfavlsVWO7KHo0RRqzbi6KVH84jFwk6jp45LjlSrW8zwHidY4bNSvonw6i+\nZTgk/VX2hY9yut/24ccdzX76Mi6eJsGeATrZR3nR7zYGX8t/C1I=\n-----END RSA PRIVATE KEY-----"
```

Comment:
You should remove this from the application, its a huge security hole to have the private key in the source code.